projects/addon-table/components/reorder/reorder.component.ts

```diff
@@ -29,6 +30,9 @@ export class TuiReorderComponent<T> {
     @Output()
     public readonly enabledChange = new EventEmitter<T[]>();
 
+    @Input()
+    public container?: PolymorpheusContent;
```

Comment:
We call these things `content`. And let's type context: `PolymorpheusContent<TuiContext<T> & { index: number }>`